src/pages/_app.tsx

```diff
@@ -21,6 +22,9 @@ declare global {
     interface Window {
         __reactRendered: boolean;
     }
+    interface Date {
+        getWeekNumber: () => number;
+    }
```

Comment:
Maybe this should actually be moved to the `polyfills.ts` module where the function is defined? I think that might make it easier for a reader to understand what's going on. 😊